frontend/src/components/sign/components/sign-in-form/sign-in-form.tsx

```diff
@@ -24,15 +39,29 @@ export const SignInForm = (): React.ReactElement => {
           Sign Up
         </Link>
       </p>
-      <form name="signinForm" onSubmit={handleSubmit} className={styles.form}>
+      <form
+        name="signinForm"
+        onSubmit={handleSubmit(onLogin)}
+        className={styles.form}
+      >
         <fieldset disabled={isLoading} className={styles.fieldset}>
-          <InputField name="Email" type="email" required={true} errors={''} />
           <InputField
-            name="Password"
+            name="email"
+            type="email"
+            required={true}
+            errors={errors}
+            control={control}
+          />
+          <InputField
+            name="password"
             type="password"
             required={true}
-            errors={'Must be at least 8 characters'}
+            errors={errors}
+            control={control}
           />
+          {error && 'data' in error && (
+            <Alert severity="error">{error.data.message}</Alert>
```

Comment:
Isn't the `data` properly already available for the `error` object?

Reply:
not my code

Reply:
yeah, but it's just typing problem. RTKQ can return some other type of error which does not have property data, so we need to check whether this property exists in data

Reply:
Ah, okay. I remembered why this check is needed. RTKQ really have 2 types of possible errors and they should be narrowed down